src/Infrastructure/Controller/Regulation/RegulationDetailController.php

```diff
@@ -36,21 +39,30 @@ public function __construct(
         requirements: ['uuid' => Requirement::UUID],
         methods: ['GET', 'POST'],
     )]
-    public function __invoke(string $uuid): Response
+    public function __invoke(Request $request, string $uuid): Response
     {
         /** @var GeneralInfoView */
         $generalInfo = $this->getRegulationOrderRecordUsing(function () use ($uuid) {
             return $this->queryBus->handle(new GetGeneralInfoQuery($uuid));
         });
 
-        /** @var RegulationOrderLocationsView */
         $regulationOrderLocations = $this->queryBus->handle(new GetRegulationLocationsQuery($uuid));
+        $featureNewLocationEnabled = $this->featureFlagService->isFeatureEnabled('loc_inversion', $request);
```

Comment:
Je dirais qu'on peut choisir l'un ou l'autre mais rester cohérent

```suggestion
        $featureLocInversionEnabled = $this->featureFlagService->isFeatureEnabled('loc_inversion', $request);
```

Ou bien

```suggestion
        $featureNewLocationEnabled = $this->featureFlagService->isFeatureEnabled('new_location', $request);
```